root_VS2012/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Framework/Js/common.js

```diff
@@ -0,0 +1,623 @@
+﻿//**********************************************************************************
+//* All Rights Reserved, Copyright (C) 2007,2012 Hitachi Solutions,Ltd.
+//**********************************************************************************
+
+// ------------------------------------------------------------
+//  イベントハンドラ
+// ------------------------------------------------------------
```

Comment:
@yusukemaegawa 

I will fix this at the same time with the all pointed out fixes.